verification/curator-service/ui/README.md

```diff
@@ -22,8 +22,24 @@ corresponding to this CSV format is available
 [here](https://docs.google.com/spreadsheets/d/1J-C7dq1rNNV8KdE1IZ-hUR6lsz7AdlvQhx6DWp36bjE).
 The following fields are currently incorporated:
 
+##### Case reference
 - `sourceEntryId`: Optional. The UUID provided by the data source to identify
 the specific case.
+
+##### Demographics
+- `gender`: Optional. One of `Female`, `Male`, `Non-binary/Third gender`, or
+`Other`.
+- `ageRangeStart`: Optional. Number (years) from 0 to 120.
+- `ageRangeEnd`: Optional. Number (years) from 0 to 120.
+- `ageRange`: Optional. Convenience field accepting an age range in the format
```

Comment:
What if they have an exact age and not a range?

Reply:
They can populate `ageRangeStart`/`ageRangeEnd`. From a CSV manipulation standpoint, that's not too much work, since the curator would just duplicate and retitle the column.

Reply:
If I were filling that out and I only filled in ageRangeStart I would assume I was filling in an open ended range e.g. 45+ rather than definitively 45. I think the safest thing to do is also having a dedicated age column, and if they want to enter an age range the form should require them to put in both boundaries on the age range (e.g. 45 and 120 for 45+). That's what we do in the single case form.

Reply:
Sorry, to be clear, filling only start or end does result in an open range. An exact age is represented by these two fields containing the same value. I agree that providing an `age` field would make the exact value scenario clear, though to also support open ranges, it'd require 4 columns for age. That's because we want to provide a convenience for the common case of the source containing a single `start-end` entry; the difficulty of getting data into this format is a consideration for bulk that we don't have to make for the single entry form.

Let's poke Aaron on this subject once he drafts his instructions.